crates/biome_grit_patterns/src/pattern_compiler/literal_compiler.rs

```diff
@@ -20,7 +21,16 @@ impl LiteralCompiler {
             AnyGritLiteral::GritBooleanLiteral(node) => Ok(Pattern::BooleanConstant(
                 BooleanConstant::new(node.value()?.text_trimmed() == "true"),
             )),
-            AnyGritLiteral::GritCodeSnippet(_) => todo!(),
+            AnyGritLiteral::GritCodeSnippet(node) => match node.source()? {
+                AnyGritCodeSnippetSource::GritBacktickSnippetLiteral(node) => {
+                    let token = node.value_token()?;
+                    let text = token.text_trimmed();
+                    let range = node.syntax().text_trimmed_range().to_byte_range();
+                    parse_snippet_content(&text[1..text.len() - 1], range, context, is_rhs)
```

Comment:
It might be worth leaving a comment explaining what we are trimming here, e.g. why there is a `1` and a `-1`. For example, I don't know, even though I can guess it by looking at the PR